renderdoccmd/3rdparty/cmdline/cmdline.h

```diff
@@ -55,7 +55,7 @@ class lexical_cast_t{
     std::stringstream ss;
     if (!(ss<<arg && ss>>ret && ss.eof()))
       throw std::bad_cast();
-    
+
```

Comment:
Please don't make whitespace only changes in third party files.

Reply:
thanks for helping review the changes, I've reverted changes in the  third party files.